portfolio/src/main/webapp/index.html

```diff
@@ -89,6 +89,12 @@ <h1>Michaela Murray's Portfolio</h1>
     <div id="inner-content">
       <div id="my-info"></div>
     </div>
+    <label id="location-label">Places I want to go:</label>
```

Comment:
I don't think you need an id on the `label` but you should add a `for` attribute that references the `locations` id.

Reply:
ah ok thanks! will update!